### PR TITLE
LPS-121404 Add missing types to remote-app-client-js get() declaration

### PR DESCRIPTION
**Pull request by wincent**
No behavior changes here; just broadening the type definition.

### CODE REVIEW

**Comment by wincent** [comment]
ci:test:sf

**Comment by wincent** [comment]
ci:test:relevant

**Comment by wincent** [comment]
@carloslancha: Here is some **TypeScript** for you to review.

you-know-you-want-to

File: modules/apps/remote-app/remote-app-client-js/src/main/resources/META-INF/resources/js/index.ts [hunk 1]
```diff
@@ -36,6 +36,20 @@ type ClientOptions = {
 	debug: boolean;
 };
 
+/**
+ * Properties accessible via the `get()` method. The remote-app-support-web
+ * module will reject requests for properties outside of this list.
+ */
+type GettableProperties =
```
Comment (jbalsas) [inline modules/apps/remote-app/remote-app-client-js/src/main/resources/META-INF/resources/js/index.ts:43]: How would one consume these definitions when writing a remote app? 

We'll need to publish these or `d.ts` files to match, I guess?

**Review by julien — APPROVED** [review]

File: modules/apps/remote-app/remote-app-client-js/src/main/resources/META-INF/resources/js/index.ts (hunk 1 above)
Comment (wincent) [reply modules/apps/remote-app/remote-app-client-js/src/main/resources/META-INF/resources/js/index.ts:43]: > We'll need to publish these or `.d.ts` files to match, I guess?

Yep.

**Comment by wincent** [comment]
ci:forward

**Pull request closed without merging**